src/lib/pipeline.ml

```diff
@@ -302,6 +306,8 @@ module Full (Bfx: Extended_edsl.Semantics) = struct
       ) in
     let normal = Stdlib.fastq_of_input parameters.normal in
     let tumor = Stdlib.fastq_of_input parameters.tumor in
+    let normal_tumor = Bfx.list [normal; tumor;] |> Bfx.concat in
+    let optitype = optitype_dna normal_tumor in
```

Comment:
should it be running on both normal and tumor DNA? seems reason to run on normal, or both, but confusing to run on both